src/components/ProfileStepper.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import firebase from 'firebase';
+import { Redirect } from 'react-router-dom';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import StepContent from '@material-ui/core/StepContent';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import ProfileInput from './ProfileInput';
+import ScholarshipInput from './ScholarshipInput';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  button: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+function getSteps() {
+  return ['Login', 'Profile Information', 'Scholarship Details'];
+}
+
+function getStepContent(step) {
+  switch (step) {
+    case 0:
+      return 'Please login to continue to the next step';
+    case 1:
+      return null;
+    case 2:
+      return null;
+    default:
+      return 'Unknown step';
+  }
+}
+
+function getStepTask(step) {
+  switch (step) {
+    case 0:
+      return null;
+    case 1:
+      return <ProfileInput />;
+    case 2:
+      return <ScholarshipInput />;
+    case 3:
+      return null;
+    default:
+      return 'Unknown step';
+  }
+}
```

Comment:
I recommend using an object for this info. E.g.:
```js
const steps = {
  0: {
    title: "step title",
    content: "step content",
    task: <StepTask />,
  },
  // etc.
}
```

Reply:
Then would `<StepTask />` be a function component?

Reply:
Ok perfect this way is much better thanks for the suggestion.
Below is how I currently how it as and need to just include task for Steps[1&2]
```javascript
const steps = {
    0: {
      title: 'Profile Selection',
      content: 'Please Select Profile Type',
      task:
        <FormControl component="fieldset">
          <FormLabel component="legend">Account Type</FormLabel>
          <RadioGroup aria-label="gender" name="gender1" value={currentUser} onChange={handleCurrentUser}>
            <FormControlLabel value="Student" control={<Radio />} label="Student" />
            <FormControlLabel value="Parent" control={<Radio />} label="Parent" />
            <FormControlLabel value="Contributor" control={<Radio />} label="Community Contributor" />
          </RadioGroup>
        </FormControl>
    },
    1: {
      title: 'Profile Information',
      content: null,
      task: null,
    },
    2: {
      title: 'Completed',
      content: null,
      task: null,
    },
  }
```